src/org/zalando/stups/friboo/system/db.clj

```diff
@@ -22,7 +22,20 @@
            (org.flywaydb.core Flyway)
            (org.postgresql.util PSQLException)
            (com.netflix.hystrix.exception HystrixBadRequestException)
-           (com.fasterxml.jackson.databind.util ISO8601Utils)))
+           (com.fasterxml.jackson.databind.util ISO8601Utils)
+           (java.util Properties)))
+
+(defn load-flyway-configuration
+  [configuration jdbc-url]
+  (let [properties (Properties.)]
+    (doseq [property configuration]
+      (when (.contains (name (key property)) "flyway")
+        (.setProperty properties (clojure.string/replace (name (key property)) "-" ".") (val property))))
```

Comment:
Yes, it's because dots in environment variables are a pain. They flyway environment will be in java.util.Properties; because the flyway function configure expects an object of the class Properties.